decbinhex4/addons/main/src/code/main_class.nim

```diff
@@ -1,89 +1,60 @@
-import gdext, gdextgen/classes/gdSceneTree, strutils, convert#convert.nim
-import gdextgen/classes / [gdInput, gdLineEdit, gdBaseButton, gdLabel]
-import gdextgen/classes / [gdWindow, gdTime]
-
-let
-  hex_letters:set[char] = HexDigits - Digits
-  nonhex_chars:set[char] = AllChars - HexDigits
-  nonbinary_numbers:set[char] = Digits - {'0', '1'}
-  dec_node:string = "grid/decimal_button"
-  bin_node:string = "grid/binary_button"
-  hex_node:string = "grid/hex_button"
-  in_node:string = "grid/inbox"
-  out_node:string = "grid/a/outlabel"
+import strutils, convert #convert.nim
+import gdext, gdextgen/classes / [gdSceneTree, gdInput, gdLineEdit, gdLabel, gdBaseButton], gdextgen/classes / [gdWindow, gdTime]
```

Comment:
Suggest either:
```nim
import gdext, gdextgen/classes / [gdSceneTree, gdInput, gdLineEdit, gdLabel, gdBaseButton, gdWindow, gdTime]
```

```nim
import gdext
import gdextgen/classes / [gdSceneTree, gdInput, gdLineEdit, gdLabel, gdBaseButton, gdWindow, gdTime]
```

Reply:
First line of imports is pure nim, second is godot with the ordering denoting importance. `[gdWindow, gdTime]` is not strictly required for the functionality of the program.